Updating CLI Homebrew formula after release (#457)
kyma-project/cli: README.md

curl -Lo kyma https://storage.googleapis.com/kyma-cli-stable/kyma-darwin # kyma-linux or kyma.exe
chmod +x kyma
```

### Post-release steps

Once the CLI release is complete, the following steps need to be performed:

> 1. Remove the existing `stable` tag and re-create it pointing to the latest commit on the `master` branch. Pushing this tag will trigger a postsubmit job that builds and pushes the latest stable binaries to a storage bucket. These are used by the continuous integration jobs.

Reviewer: Please remove the `stable` step as we decided to do it independently of the release.

> 2. Update the Homebrew formula of the CLI to the just-released version by following this [guide](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md). For a sample Homebrew Kyma CLI formula version bump see the information in [this](https://github.com/Homebrew/homebrew-core/pull/52375) PR.

Reviewer: ```suggestion
2. If the Homebrew formula of the CLI does not get updated by the Homebrew team within three days of the release, manually update the formula of the CLI to the just-released version by following this [guide](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md). For a sample Homebrew Kyma CLI formula version bump see the information in [this](https://github.com/Homebrew/homebrew-core/pull/52375) PR.
```

curl -Lo kyma https://storage.googleapis.com/kyma-cli-stable/kyma-darwin # kyma-linux or kyma.exe
chmod +x kyma
```

### Kyma CLI Homebrew formula

> If the Homebrew formula of the CLI does not get updated by the Homebrew team within three days of the release, then manually update the formula of the CLI to the just-released version by following this [guide](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md). For a sample Homebrew Kyma CLI formula version bump see the information in [this](https://github.com/Homebrew/homebrew-core/pull/52375) PR.

Reviewer 2: ```suggestion
If the Homebrew formula of the CLI does not get updated by the Homebrew team within three days of the release, update the formula of the CLI manually to the most recent version by following this [guide](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md). For a sample Homebrew Kyma CLI formula version bump, see [this](https://github.com/Homebrew/homebrew-core/pull/52375) PR.
```
